_config.yml

```diff
@@ -9,6 +9,7 @@ kramdown:
 repository: opendocsg/opendoc-theme
 server_PROD: https://search.opendoc.sg
 server_DEV: https://search-staging.opendoc.sg
+PDF_storage_URL: https://opendoc-theme-pdf.s3-ap-southeast-1.amazonaws.com/opendoc-theme
```

Comment:
can we leave this as default

Reply:
Yeah, this var will be ignored if either PDF_GEN_API_KEY or PDF_GEN_API_SERVER environment variables is not set. By right each repo should have a different pathname, otherwise they will overwrite each other's PDFs.